uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1000.php

```diff
@@ -36,11 +36,20 @@ private function up1000_updateHalfDayPmLabel() {
    * @param string $label
    */
   private function up1000_updateOptionValueLabel($name, $label) {
-    civicrm_api3('OptionValue', 'get', [
-      'sequential' => 1,
-      'option_group_id' => 'hrleaveandabsences_leave_request_day_type',
-      'name' => ['IN' => [$name]],
-      'api.OptionValue.create' => ['id' => '$value.id', 'label' => $label],
-    ]);
+    try {
+      civicrm_api3('OptionValue', 'get', [
```

Comment:
@davialexandre can you explain more in the description why this is required? I'm looking at #1998 for @tunbola but because of the order of method calls in the upgrader (first import XML, then run upgraders) I don't see how the upgrader could get here without the option group existing.